scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,383 @@
+        android_global_config.set_config_value(key=android_support.SETTINGS_SIGNING_STORE_PASSWORD.key, value=auto_store_pwd, validate_value=False)
+    
+    if not auto_key_pwd:
+        android_global_config.set_password(android_support.SETTINGS_SIGNING_KEY_PASSWORD.key)
+    else:
+        android_global_config.set_config_value(key=android_support.SETTINGS_SIGNING_KEY_PASSWORD.key, value=auto_key_pwd, validate_value=False)
+
+def deloy_to_android_device(target_android_project_path: pathlib.Path,
+                            build_config:str,
+                            project_name:str,
+                            android_sdk_home:pathlib.Path,
+                            org_name:str|None = None,
+                            activity_name:str|None = None):
+    is_pc = exp.get_default_asset_platform() == 'pc'
+
+    adb_exec_path = android_sdk_home / ('platform-tools/adb' + ('.exe' if is_pc else ''))
+    exp.process_command([adb_exec_path, 'install', '-t', '-r', target_android_project_path / f'app/build/outputs/apk/{build_config}/app-{build_config}.apk'])
+
+    time.sleep(1)
+
+    if not activity_name:
+        activity_name = f'{project_name}Activity'
+    
+    if not org_name:
+        org_name = f'org.o3de.{project_name}'
+
+    exp.process_command([adb_exec_path, 'shell', 'am', 'start', '-a', 'android.intent.action.MAIN','-n', f'{org_name}/.{activity_name}'])
+
+
+def export_source_android_project(ctx: exp.O3DEScriptExportContext,
+                           target_android_project_path: pathlib.Path,
+                           seedlist_paths: List[pathlib.Path],
+                           seedfile_paths: List[pathlib.Path],
+                           level_names: List[str],
+                           asset_pack_mode: str = 'PAK',
+                           engine_centric: bool = False,
+                           should_build_tools: bool = True,
+                           should_build_all_assets: bool = True,
+                           build_config: str = 'profile',
+                           tool_config: str = 'profile',
+                           tools_build_path: pathlib.Path | None =None,
+                           max_bundle_size: int = 2048,
+                           fail_on_asset_errors: bool = False,
+                           deploy_to_device: bool = False,
+                           org_name: str|None = None,
+                           activity_name:str|None = None,
+                           logger: logging.Logger|None = None) -> None:
+    
+    expected_platform = 'android'
+    project_path = ctx.project_path
+    engine_path = ctx.engine_path
+    project_name = ctx.project_name
+
+    if not logger:
+        logger = logging.getLogger()
+        logger.setLevel(logging.ERROR)
+
+    is_installer_sdk = manifest.is_sdk_engine(engine_path=engine_path)
+
+    is_pc = exp.get_default_asset_platform() == 'pc'
+
+    o3de_cli_script_path = engine_path / ('scripts/o3de' + ('.bat' if is_pc else '.sh'))
```

Comment:
`O3DE_SCRIPT_EXTENSIONS` is already defined [here](https://github.com/o3de/o3de/blob/8fc008743313813933cd0be190535fc4ed79a92b/scripts/o3de/o3de/android_support.py#L37-L59)